doc/api/tls.md

```diff
@@ -1094,6 +1094,38 @@ See
 [SSL_get_shared_sigalgs](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_shared_sigalgs.html)
 for more information.
 
+### `tlsSocket.exportKeyingMaterial(length, label[, context])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `length` {number} number of bytes to retrieve from keying material
+* `label` {string} an application specific label, typically this will be a
+value from the IANA Exporter Label Registry
+(https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#exporter-labels).
```

Comment:
Optional suggestion: Replace the long URL with an actual link (instead of showing the URL), allowing readers to just click on `IANA Exporter Label Registry`.